upgrade_analysis/wizards/upgrade_install_wizard.py

```diff
@@ -34,7 +34,7 @@ class UpgradeInstallWizard(models.TransientModel):
     def _module_ids_domain(self, extra_domain=None):
         domain = [
             "&",
-            ("state", "not in", ["uninstallable", "unknown"]),
+            ("state", "not in", ["installed", "uninstallable", "unknown"]),
```

Comment:
I'm not sure to understand. how do you run the analysis of the ``base`` module, that is allways installed.

Reply:
This is for reinstalling things, so no need to reinstall what is already installed.